Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:lts AS frontend
+RUN mkdir /app
+WORKDIR /app
+ADD ./frontend/ /app/
+RUN yarn install
+RUN yarn build
+
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.6 AS backend
+WORKDIR /app
+COPY ./backend/ /app/
+RUN pip install -r requirements.txt
+COPY --from=frontend /app/build/ /app/static/
```

Comment:
Hey, buildmaster! The way it's configured now, the `index.html` produced by builds needs to go to `backend/templates`. I'm not sure if we can make it work from the same place where we're serving static files, but we can look into it, or you could tweak the Dockerfile to move the file to templates! 😇